bindings/go/src/fdb/tuple/tuple.go

```diff
@@ -249,6 +292,18 @@ func (p *packer) encodeUUID(u UUID) {
 	p.putBytes(u[:])
 }
 
+func (p *packer) encodeVersionstamp(v Versionstamp) {
+	p.putByte(versionstampCode)
+
+	if p.versionstampPos != -1 && v.TransactionVersion == incompleteTransactionVersion {
+		panic(fmt.Sprintf("Tuple can only contain one incomplete versionstamp"))
+	} else {
+		p.versionstampPos = int32(len(p.buf))
```

Comment:
I think this will set the versionstamp position even if the versionstamp is complete

Reply:
Now that I'm looking at it, that seems obviously incorrect, which makes me very skeptical my testing setup is correct. I thought I had resolved it given I could force it to fail. Will keep investigating